### PR TITLE
feat: add new tokens

### PR DESCRIPTION
**Pull request by barathcj**
Ticket: BG-28921

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -283,7 +289,7 @@ export const coins = CoinMap.fromCoins([
   erc20('hydro', 'Hydro', 18, '0xebbdf302c940c6bfd49c6b165f457fdb324649bc', UnderlyingAsset.HYDRO),
   erc20('i8', 'i8 Exchange Token', 18, '0xb61bb66bf9caba531e6bb2fb75dec389a1664dfd', UnderlyingAsset.I8),
   erc20('idex', 'IDEX', 18, '0xb705268213d593b8fd88d3fdeff93aff5cbdcfae', UnderlyingAsset.IDEX),
-  erc20('idrc', 'Idrc', 18, '0x569f2241551011d5710c40fc61e0b3906c780b2f', UnderlyingAsset.IDRC),
+  erc20('idrc', 'Rupiah Coin', 18, '0x569f2241551011d5710c40fc61e0b3906c780b2f', UnderlyingAsset.IDRC),
```
Comment (wesgraham) [inline modules/statics/src/coins.ts:292]: Confirming this name change is intentional

**Review by wesgraham — DISMISSED** [review]
lgtm

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -132,6 +132,7 @@ export const coins = CoinMap.fromCoins([
   tofc('ofctbtg', 'Test Bitcoin Gold', 8, UnderlyingAsset.BTG, CoinKind.CRYPTO),
   tofc('ofctbsv', 'Test Bitcoin SV', 8, UnderlyingAsset.BSV, CoinKind.CRYPTO),
   tofc('ofcteos', 'Test Eos', 4, UnderlyingAsset.EOS, CoinKind.CRYPTO),
+  erc20('1inch', '1inch Token', 18, '0x111111111117dc0aa78b770fa6a738034120c302', UnderlyingAsset['1inch']),
```
Comment (wesgraham) [inline modules/statics/src/coins.ts:135]: Looks like this is failing test cases

**Review by wesgraham — APPROVED** [review]

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (barathcj) [reply modules/statics/src/coins.ts:292]: yes

**Pull request merged**